### PR TITLE
Rh tabbed interface

### PR DESCRIPTION
**Pull request by reherr**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
 
     respond_to do |format|
       if @comment.save
-        format.html { redirect_to comment_url(@comment), notice: "Comment was successfully created." }
+        format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
```
Comment (borvux) [inline app/controllers/comments_controller.rb:29]: good job on noticing the change

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  def show
+    @user = User.find_by!(username: params.fetch(:username))
+  end
+
+  def liked
+    @user = User.find_by!(username: params.fetch(:username))
+  end
+
+  def feed
+    @user = current_user
```
Comment (borvux) [inline app/controllers/users_controller.rb:11]: good job using the current_user variable

File: app/views/comments/_form.html.erb [hunk 3]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :body, style: "display: block" %>
-    <%= form.text_area :body %>
+  <div class="mb-3">
+    <%= form.text_area :body, class: "form-control" %>
   </div>
 
   <div>
-    <%= form.submit %>
+    <%= form.submit class: "btn btn-outline-secondary btn-block" %>
```
Comment (borvux) [inline app/views/comments/_form.html.erb:21]: great job refactoring the code

File: app/views/photos/_photo.html.erb [hunk 4]
```diff
@@ -1,27 +1,29 @@
...
+              </div>
+            </div>
+          </li>
+        <% end %>
+      </ul>
+      <div class="card-body">
+        <%= render "comments/form", comment: photo.comments.build %>
+      </div>
+    </div>
+  </div>
```
Comment (borvux) [inline app/views/photos/_photo.html.erb:28]: nice job on making it look nicer with bootstrap

File: app/views/users/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+      </li>
+      <li class="nav-item">
+        <%= link_to "Followers", followers_path(@user.username), class: "nav-link" %>
+      </li>
+      <li class="nav-item">
+        <%= link_to "Following", following_path(@user.username), class: "nav-link" %>
+      </li>
+    </ul>
+  </div>
+</div>
```
Comment (borvux) [inline app/views/users/show.html.erb:25]: could potentially have all of the nav links in a partials but its okay !

File: config/routes.rb [hunk 6]
```diff
@@ -8,8 +8,12 @@
...
   resources :likes
   resources :photos
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  get ":username/liked" => "users#liked", as: :liked
+  get ":username/feed" => "users#feed", as: :feed 
+  get ":username/followers" => "users#followers", as: :followers
+  get ":username/following" => "users#following", as: :following
```
Comment (borvux) [inline config/routes.rb:15]: good job adding the custom routes

**Review by borvux — APPROVED** [review]

**Review by CarlosCastanedaDev — APPROVED** [review]
Looks great, Rebecca!

In your "users" view folder you could also create a partial for all the tab info and use the partial in the feed, followers, following, liked and show view files.

**Pull request merged**